lib/addons/istio-addons/istio-cni.ts

```diff
@@ -0,0 +1,27 @@
+import { Construct } from 'constructs';
+import { ClusterInfo } from "../../spi";
+import { HelmAddOn, HelmAddOnProps } from "../helm-addon";
+import { dependable, supportsALL } from '../../utils';
+
+const defaultProps: HelmAddOnProps = {
+    name: 'istio-cni',
+    release: 'cni',
+    namespace: 'istio-system',
+    chart: 'cni',
+    version: "1.20.1",
```

Comment:
let's have a central point for the version. e.g. `export const VERSION` and use that in each istio addon when setting version in the defaultProps, so that we can set it once for all, but customers can still override it per addon.

Reply:
Fixed